WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -3,6 +3,67 @@ local L = WeakAuras.L;
 
 WeakAuras.regionPrototype = {};
 
+-- Adjusted Duration
+
+function WeakAuras.regionPrototype.AddAdjustedDurationToDefault(default)
+  default.useAdjustededMax = false;
+  default.useAdjustededMin = false;
+end
+
+function WeakAuras.regionPrototype.AddAdjustedDurationOptions(options, data, order)
+  options.useAdjustededMin = {
+    type = "toggle",
+    name = L["Set Minimum Value/Duration"],
+    desc = L["Values/Remaining Time below this value are displayed as no progress."],
+    order = order
+  };
+
+  options.adjustedMin = {
+    type = "range",
+    min = 0,
+    softMax = 300,
```

Comment:
IIRC some classes have a maximum of 180 nowadays. I reduced it to 200 and added the bigStep.